src/EditorFeatures/TestUtilities/Workspaces/TestWorkspace_XmlConsumption.cs

```diff
@@ -71,12 +71,13 @@ public static TestWorkspace Create(string xmlDefinition, bool completed = true,
             return Create(workspaceElement, completed, openDocuments, exportProvider, workspaceKind);
         }
 
-        public static TestWorkspace Create(
+        internal static TestWorkspace Create(
```

Comment:
why change this?

Reply:
➡️ The type of the new parameter has internal accessibility, which introduces a compile error if this doesn't change at the same time.